src/cr/cube/cube_slice.py

```diff
@@ -0,0 +1,145 @@
+'''Home of the CubeSlice class.'''
+
+
+class CubeSlice(object):
+    '''Implementation of CubeSlice class.
+
+    The CubeSlice is used to uniformly represent all cubes as 2D tables.
+    For 1D cubes, this is achieved by inflating. For 3D cubes, this is
+    achieved by slicing.
+    '''
+
+    def __init__(self, cube, index):
+        self._cube = cube
+        self._index = index
+
+    def _update_args(self, kwargs):
```

Comment:
I would inline this and the next function. They're only called from one place. This would potentially make the logic in `_call_cube_method` clearer.